src/test/java/org/snomed/snowstorm/core/data/services/ConceptServiceTest.java

```diff
@@ -643,6 +643,44 @@ void testCreateDataAttribute() throws ServiceException {
 		assertEquals(PRIMITIVE, axiom.getDefinitionStatusId());
 	}
 
+	@Test
+	public void testCreateConceptWithConcreteIntValue() throws ServiceException {
+		//given
+		final Relationship relationship = new Relationship("200001001", 20170131, true, "900000000000012004", "900000000000441003", "#1", 0, "116680003", "900000000000011006", "900000000000451002");
```

Comment:
@kaicode, we spoke about Relationships coming from Axioms not having an id. Whenever I remove the id from the test Relationship, the test fails because the classExpressionType isn't as expected. I'm looking into this.

Reply:
Ignore this comment; all working now.